layers/+music/spotify/packages.el

```diff
@@ -35,12 +35,16 @@
   (use-package counsel-spotify
     :defer t
     :init (progn
+            (spacemacs/declare-prefix "amss" "counsel-spotify-search")
+            (spacemacs/declare-prefix "amssT" "search-tracks-by...")
             (spacemacs/set-leader-keys
               "amssa" 'counsel-spotify-search-artist
               "amssA" 'counsel-spotify-search-album
               "amsst" 'counsel-spotify-search-track
               "amssTa" 'counsel-spotify-search-tracks-by-artist
-              "amssTA" 'counsel-spotify-search-tracks-by-album))))
-
-(defun spotify/post-init-counsel-spotify ()
-  (load-library "counsel-spotify"))
+              "amssTA" 'counsel-spotify-search-tracks-by-album))
+    :commands (counsel-spotify-search-artist
```

Comment:
Nitpick: Usually we put `:commands` above `:init` and `:config`.

Reply:
I can see why that makes sense